frontend/packages/kubevirt-plugin/integration-tests/tests/models/rhvImportWizard.ts

```diff
@@ -99,9 +91,13 @@ export class RhvImportWizard extends ImportWizard {
     await this.selectSourceVirtualMachine(sourceVMName);
     await this.waitForSpinner();
     // Clicking `edit` button to reach network and storage settings
+    await browser.sleep(2000);
```

Comment:
let's not use hard timeout here, but wait for the element to be clickable

Reply:
How I can check that element is clickable?

Reply:
you can use a combination of these https://www.protractortest.org/#/api?view=ProtractorExpectedConditions . Sometimes it is necessary to scroll the elem into view as well

Reply:
you can also get inspired in `./frontend/packages/dev-console/integration-tests/utilities/elementInteractions.ts` . eg `selectCheckBox`